doc/query.md

```diff
@@ -39,25 +39,30 @@ Secondly, these documents are in the datastore:
 ```java
 { "name": "mike", 
   "age": 12, 
-  "pet": {"species": "cat"} };
+  "pet": {"species": "cat"},
+  "comment": "Mike goes to middle school and likes reading books." };
 
 { "name": "mike", 
   "age": 34, 
-  "pet": {"species": "dog"} };
+  "pet": {"species": "dog"},
+  "comment": "Mike is a doctor and likes reading books." };
 
 { "name": "fred", 
   "age": 23, 
-  "pet": {"species": "cat"} };
+  "pet": {"species": "cat"},
+  "comment": "Fred works for a startup out of his home office." };
 ```
 
-### Creating indexes
+### Indexes
 
 In order to query documents, indexes need to be created over
 the fields to be queried against.
 
-Use `ensureIndexed(List<Object> fieldNames, String indexName)` to create indexes. These indexes are persistent across application restarts as they are saved to disk. They are kept up to date documents change; there's no need to call `ensureIndexed(List<Object> fieldNames, String indexName)` each time your applications starts, though there is no harm in doing so.
+#### Managing JSON indexes
 
-The first argument to `ensureIndexed(List<Object> fieldNames, String indexName)` is a list of fields to put into this index. The second argument is a name for the index. This is used to delete indexes at a later stage and appears when you list the indexes in the database.
+Use `ensureIndexed(List<Object> fieldNames, String indexName)` to create standard JSON based indexes. These indexes are persistent across application restarts as they are saved to disk. They are kept up to date as documents change; there's no need to call `ensureIndexed(List<Object> fieldNames, String indexName)` each time your applications starts, though there is no harm in doing so.
```

Comment:
I think this is implied by the statement earlier in the doc `The IndexManager object provides the ability to manage query indexes and execute queries`